packages/subgraph/tests/dao/dao_v1_0_0.test.ts

```diff
@@ -222,123 +243,135 @@ describe('handleDeposited: ', () => {
     let transferId = getTransferId(txHash, logIndex, 0);
 
     // check ERC20Contract entity
-    eq('ERC20Contract', tokenId, 'id', tokenId);
-    eq('ERC20Contract', tokenId, 'name', 'DAO Token');
-    eq('ERC20Contract', tokenId, 'symbol', 'DAOT');
-    eq('ERC20Contract', tokenId, 'decimals', '6');
+    daoTokenContract.assertEntity();
     assert.entityCount('ERC20Contract', 1);
 
     // check ERC20Balance entity
-    eq('ERC20Balance', balanceId, 'id', balanceId);
-    eq('ERC20Balance', balanceId, 'token', tokenId);
-    eq('ERC20Balance', balanceId, 'dao', daoId);
-    eq('ERC20Balance', balanceId, 'balance', ERC20_AMOUNT_HALF);
-    eq('ERC20Balance', balanceId, 'lastUpdated', timestamp.toString());
+    let erc20Balance = new ExtendedERC20Balance().withDefaultValues();
+    // expected changes
+    let balance = BigInt.fromString(ERC20_AMOUNT_HALF);
+    erc20Balance.balance = balance;
+    erc20Balance.lastUpdated = timestamp;
+    // assert
+    erc20Balance.assertEntity();
     assert.entityCount('ERC20Contract', 1);
 
     // Check ERC20Transfer
-    let from = newEvent.params.sender.toHexString();
-    eq('ERC20Transfer', transferId, 'id', transferId);
-    eq('ERC20Transfer', transferId, 'dao', daoId);
-    eq('ERC20Transfer', transferId, 'token', tokenId);
-    eq('ERC20Transfer', transferId, 'amount', ERC20_AMOUNT_HALF);
-    eq('ERC20Transfer', transferId, 'from', from);
-    eq('ERC20Transfer', transferId, 'to', DAO_ADDRESS);
-    eq('ERC20Transfer', transferId, 'type', 'Deposit');
-    eq('ERC20Transfer', transferId, 'txHash', txHash.toHexString());
-    eq('ERC20Transfer', transferId, 'createdAt', timestamp.toString());
+    let erc20Transfer = new ExtendedERC20Transfer().withDefaultValue(
+      transferId
+    );
+    // expected changes
+    erc20Transfer.amount = balance;
+    erc20Transfer.txHash = txHash;
+    // assert
+    erc20Transfer.assertEntity(true);
     assert.entityCount('ERC20Transfer', 1);
   });
 
   test('ERC20: creates multiple events and updates balance', () => {
-    let newEvent = createNewDepositedEvent(
+    let dao = new ExtendedDao().withDefaultValues();
+
+    let newEvent = dao.createEvent_Deposited(
       ADDRESS_ONE,
-      DAO_TOKEN_ADDRESS,
       ERC20_AMOUNT_HALF,
-      STRING_DATA,
-      DAO_ADDRESS
+      STRING_DATA
     );
 
     handleDeposited(newEvent);
-    eq('ERC20Balance', balanceId, 'balance', ERC20_AMOUNT_HALF);
 
-    getBalanceOf(DAO_TOKEN_ADDRESS, DAO_ADDRESS, ERC20_AMOUNT_FULL);
+    // check ERC20Balance entity
+    let erc20Balance = new ExtendedERC20Balance().withDefaultValues();
+    // expected changes
+    let balance = BigInt.fromString(ERC20_AMOUNT_HALF);
+    erc20Balance.balance = balance;
+    erc20Balance.lastUpdated = newEvent.block.timestamp;
+    // assert
+    erc20Balance.assertEntity();
+
+    // getBalanceOf(DAO_TOKEN_ADDRESS, DAO_ADDRESS, ERC20_AMOUNT_FULL);
```

Comment:
Do this comment have a purpose?

Reply:
thanks for spotting that,
yes, it should not be there.